.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Run tests
+on:
+  push:
+  pull_request:
+    branches: [ master ]
```

Comment:
This is to trigger CI only on pull requests to `master`, right? If so I think we can potentially avoid this restriction and just run GH actions on any PR against any branch.

Reply:
You're right. I removed the limitation